### PR TITLE
Fix rpc call receiverd from ws

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: src/ws.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ module.exports = (routing, server, console) => {
       const { name, method, args = [] } = obj;
       const entity = routing[name];
       if (!entity) return connection.send('"Not found"', { binary: false });
-      const handler = entity[method];
+      const exp = entity[method];
+      const handler = typeof exp === 'object' ? exp.method : exp;
```
Comment (nechaido) [inline src/ws.js:16]: Shouldn't this be handled by the loader?
So that it always places the handler into exp.method?

Comment (tshemsedinov) [reply src/ws.js:16]: It's a good idea to do it here, so one if-statement execution is better than checking this at all rpc calls

Comment (tshemsedinov) [reply src/ws.js]: See latest fixup

File: src/ws.js [hunk 2]
```diff
@@ -12,7 +12,8 @@ module.exports = (routing, server, console) => {
       const { name, method, args = [] } = obj;
       const entity = routing[name];
       if (!entity) return connection.send('"Not found"', { binary: false });
-      const handler = entity[method];
+      const exported = entity[method];
+      const handler = exported?.method;
```
Comment (nechaido) [inline src/ws.js:16]:
```suggestion
      const handler = entity[method]?.method;
```

**Review by nechaido — APPROVED** [review]
LGTM otherwise

**Pull request merged**